crawl-ref/source/dat/strings/de/hiscores.txt

```diff
@@ -671,25 +671,25 @@ Zu Tode geknabbert durch Software-Bugs
 %%%%
 # %s could be a monster or a thing
 ... while paralysed by %s
-... während der Lähmung von {dat}%s
+... während einer Lähmung durch {dat}%s
```

Comment:
Shouldn't this be "von"?
For example, "... während einer Lähmung von einer Hornisse"

Reply:
This example should be:
"... während einer Lähmung durch eine Hornisse"